src/command-line/index.js

```diff
@@ -7,7 +7,12 @@ var fsextra = require("fs-extra");
 var path = require("path");
 var Helper = require("../helper");
 
-program.version(pkg.version, "-v, --version");
+program.version({
+	// Hack to defer execution of Helper.getGitCommit()
+	toString() {
+		return Helper.getGitCommit() ? Helper.getGitCommit() : pkg.version;
+	}
```

Comment:
Are you trying to check if `Helper.getGitCommit` exists? Because you are invoking it either way.